contact.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <link rel="stylesheet" href="css/style.css">
+  <title>Contact</title>
+  <script>
+    function initMap() {
+      const garneau = { lat: 46.7935879, lng: -71.2627805 };
+      const map = new google.maps.Map(document.getElementById("map"), {
+        zoom: 12,
+        center: garneau,
+      });
+      const marker = new google.maps.Marker({
+        position: garneau,
+        map: map,
+      });
+    }
+    window.initMap = initMap;
+  </script>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-lg bg-body-tertiary">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="index.html">LOGO</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup"
+          aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+            <a class="nav-link" href="index.html">Accueil</a>
+            <a class="nav-link" href="apropos.html">À propos</a>
+            <a class="nav-link" href="mesprojets.html">Mes Projets</a>
+            <a class="nav-link active" aria-current="page" href="contact.html">Contact</a>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <nav class="mt-2" style="--bs-breadcrumb-divider: '>';" aria-label="breadcrumb">
+      <ol class="breadcrumb mx-2">
+        <li class="breadcrumb-item"><a class="text-success" href="index.html">Accueil</a></li>
+        <li class="breadcrumb-item active" aria-current="page">À propos</li>
+      </ol>
+    </nav>
+    <h1 class="text-success text-center mt-5 mb-5 fs-3">Contact</h1>
+    <div class="m-auto text-center d-flex justify-content-between" style="max-width: 600px;">
+      <a class="mx-3" href="#"><img src="assets/facebook.png" alt="Facebook" style="width: 50px;"></a>
```

Comment:
Your links are empty. Also, try to use the `target='_blank'` properties for your links that make reference to external pages.

Reply:
Done.